x/loan/keeper/msg_server_request_loan.go

```diff
@@ -22,7 +26,20 @@ func (k msgServer) RequestLoan(goCtx context.Context, msg *types.MsgRequestLoan)
 
 	// TODO: collateral has to be more than the amount (+fee?)
 
-	// moduleAcc := sdk.AccAddress(crypto.AddressHash([]byte(types.ModuleName)))
+	// Get Ratio from liquditiy pool
+	loanCoin, err := sdk.ParseCoinNormalized(msg.Amount)
+	collateralCoin, err := sdk.ParseCoinNormalized(msg.Collateral)
+
+	denomA, denomB := liquiditytypes.AlphabeticalDenomPair(loanCoin.Denom, collateralCoin.Denom)
+	coinDenoms := []string{denomA, denomB}
+
+	poolName := liquiditytypes.PoolName(coinDenoms, 1)
+	poolCoinDenom := liquiditytypes.GetPoolCoinDenom(poolName)
+	poolReserveAcc := liquiditytypes.GetPoolReserveAcc(poolCoinDenom, true)
```

Comment:
```suggestion
	poolReserveAcc := liquiditytypes.GetPoolReserveAcc(poolName, false)
```

Hi, liquidity v1.4.0 currently using the length of `ReserveAcc` as 20bytes. Therefore, it is right to Get PoolReserveAcc as `false` and hand over `PoolName` not `PoolCoinDenom`.

I couldn't check because there was no test code for `RequestLoan`, but please try it and let me know if there's any other problem.